### PR TITLE
Rf test me14 18

### PR DESCRIPTION
**Pull request by EdwinFairchild**
image

### CODE REVIEW

File: Examples/MAX32655/RF_Test/CLI-commands.c [hunk 1]
```diff
@@ -489,14 +493,23 @@ static BaseType_t cmd_ConstTx(char *pcWriteBuffer, size_t xWriteBufferLen,
...
-        /* Enable constant TX */
-        MXC_R_TX_CTRL = 0x1;
-
-        /* Enable pattern generator, set PRBS-9 */
-        MXC_R_CONST_OUPUT = 0x0;
-        MXC_R_PATTERN_GEN = 0x4B;
+            /* Enable pattern generator, set PRBS-9 */
+            MXC_R_CONST_OUPUT = 0x0;
+            MXC_R_PATTERN_GEN = 0x4B;
+        }
```
Comment (EdwinFairchild) [inline Examples/MAX32655/RF_Test/CLI-commands.c:510]: optionally running 2 versions of the test for debugging.
Using PalBbBleEnablePrbs15 generates the large waveforms in Universal Radio Hacker (URH)
and the smaller waveforms are generated to the way it was being done originally. 
I thought it was a power discrepency so I changed power to 4dBm but still the new version is significantly different.
I gather one version is right or better, than the other.

**Comment by EdwinFairchild** [comment]
@kevin-gillespie  Just for  clarity , do you want the constTx option removed and  have user use prsb15 from the `tx` test option. Or do you want the constTx option to use the prsb15 method.? 
image

**Comment by EdwinFairchild** [comment]
channels and power levels tested all 3 devices
Requires pump in RF phy version to use the newly implemented PRBS15

**Review by yc-adi — APPROVED** [review]

**Pull request merged**